### ISSUE TITLE
failed to run the command: from colored import fg, bg, attr

### ISSUE THREAD

**Issue by asmwarrior** [pyscripter/pyscripter#1106]
When I run the command inside the PyScripter's internal "Python Interpreter", I got such error: 

~~~~

>>> from colored import fg, bg, attr
Traceback (most recent call last):
  File "<interactive input>", line 1, in <module>
  File "E:\download\sci\Winpython64-3.8.8.0\WPy64-3880\python-3.8.8.amd64\lib\site-packages\colored\__init__.py", line 7, in <module>
    from .colored import *
  File "E:\download\sci\Winpython64-3.8.8.0\WPy64-3880\python-3.8.8.amd64\lib\site-packages\colored\colored.py", line 11, in <module>
    IS_TTY = sys.stdout.isatty() and sys.stderr.isatty()
AttributeError: 'AsyncStream' object has no attribute 'isatty'

~~~~

Can any one knows which cause this issue?

I can run those command without any issue in the Python's command line prompt window.

I'm using WinPython 3.8.8.0 under Windows7. Thanks.

**Comment by pyscripter** [comment]
interpreter in PyScripter and other Python IDEs (IDLE etc.) is GUI based and not a real console.  Units like colored only work with a real console.  So you cannot use them to color the PyScripter interpreter output I am afraid.

With PyScripter 4.0.0 (see https://groups.google.com/g/pyscripter/c/Xe6DI4qtNBc) the program will run without errors but the output will not be colored.
![image](https://user-images.githubusercontent.com/1311616/114968575-52ac7080-9e7f-11eb-846d-00630471c961.png)

**Comment by asmwarrior** [comment]
First, thanks for the reply.

I just tried your PyScripter 4.0.0, it works OK, see the image shot below

![PyScripter4 0 0](https://user-images.githubusercontent.com/561818/114993237-3126a580-9ece-11eb-8a0d-560d5019a0f5.png)

And I tried in PyScripter 3.8.6 again, it still failed, see below:
![PyScripter3 8 6](https://user-images.githubusercontent.com/561818/114993312-456aa280-9ece-11eb-8652-27eb2d1c5969.png)

And I have tried in other IDEs, such as IDLE, Spyder, Thonny, all works OK, at least when I type the command `from colored import fg, bg, attr`, they don't report errors.

So, I think it is the issue inside the PyScripter 3.8.6.

Any way, thanks for your work on the PyScripter! I will try to use the 4.0.0 version, and see if I can report some issues.
